index.html

```diff
@@ -259,5 +259,45 @@ <h1>
         </div>
       </section>
     </main>
+    <footer>
+      <!-- contact form -->
+      <div class="contact-container">
+        <form
+          action="https://formspree.io/f/mpznvrqy"
+          method="post"
+          class="container"
+        >
+          <h2>Contact form</h2>
+          <p class="footer-paragraph">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d
+            <br />love to help with it
+          </p>
+          <div class="form-elements">
+            <input type="text" placeholder="Yeremias" maxlength="30" required />
+          </div>
+          <div class="form-elements">
+            <input type="email" placeholder="natajayanj14@gmail.com" required />
+          </div>
+          <div>
+            <textarea
+              class="form-elements"
+              name=""
+              placeholder="Write your message here"
+              id=""
+              maxlength="500"
+            ></textarea>
```

Comment:
- [ ]  As stated the project requirement A user cannot submit the form if one or more of the fields is empty, So Kindly I recommend you to add for message input fields to have the `required` attribute to activate the validation on submit.
- [ ] Kindly make sure that the form submitted successfully, because when I submitted the form it show me below error message it's because of you didn't specify `name` and `id` attribute for text-area type of the message, So Kindly I recommend you to add `name=message` and `id=message`.

![Capture](https://user-images.githubusercontent.com/93598189/206436359-7736b4a7-d126-4b26-a3f1-c7366a692caf.PNG)

Reply:
Hi @Abenezer-Tilahun,
I have added the name, required and id attributes to the form elements as suggested.
Thank you for reviewing my PR.